Sharpmake/Project.cs

```diff
@@ -2545,4 +2545,98 @@ public AndroidPackageProject(Type targetType) : base(targetType)
         {
         }
     }
+
+    /// <summary>
+    /// A Rust project.
+    /// 
+    /// Requires the cargo build tools and cbindgen to be installed.
+    /// </summary>
```

Comment:
Please move your Rust related stuff to RustProject.cs

Project.cs contains to much stuff and we need to split it.